plugins/backend/local/libmrss/mrss_parser.c

```diff
@@ -365,7 +365,7 @@ __mrss_parser_atom_entry (nxml_t * doc, nxml_data_t * cur, mrss_t * data)
 	  {
 	  	if (item->description)
 	  	{
-	  	  nxml_free(item->description);
+	  	  free(item->description);
```

Comment:
If you look in nxmle_get_string(), it calls nxml_get_string(), which creates the string with `strdup()`, so `free` is the right thing to call here, not `nxmle_free`.